src/components/errorbars/plot.js

```diff
@@ -13,16 +13,13 @@ var d3 = require('d3');
 var isNumeric = require('fast-isnumeric');
 
 var subTypes = require('../../traces/scatter/subtypes');
-var styleError = require('./style');
-
 
 module.exports = function plot(traces, plotinfo, transitionConfig) {
     var isNew;
     var xa = plotinfo.x(),
         ya = plotinfo.y();
 
-    transitionConfig = transitionConfig || {};
-    var hasAnimation = isNumeric(transitionConfig.duration) && transitionConfig.duration > 0;
+    var hasAnimation = transitionConfig && transitionConfig.duration > 0;
```

Comment:
In other words, `=== 0` has no special behavior but uses internal optimizations that speed things up.